.gitignore

```diff
@@ -1,2 +1,19 @@
-_build
+# backup files
+*~
+*.bak
+.DS_Store
+
+# project file
 *.hdp
+
+# documentation build directory
+build/
+
+# compiler build directory
+_build/
+
+# dylan tool package cache
+_packages/
+
+# package registry folder
+registry/
```

Comment:
Leading `/` for all three of the above also?

Reply:
Same comment as above